documentation/autoinstall-reference.md

```diff
@@ -309,7 +322,7 @@ When using the "lvm" layout, LUKS encryption can be enabled by supplying a passw
         name: lvm
         password: LUKS_PASSPHRASE
 
-The default is to use the lvm layout.
+The default is to use the `lvm` layout.
 
 #### sizing-policy
```

Comment:
I guess we want to capitalize "Sizing" as well.

Reply:
Fixed, thanks.